practice/practice-02/src/styles/main.scss

```diff
@@ -1 +1,20 @@
-@import "./base/reset.scss";
+// Abstracts
+@import "./abstracts/mixins";
+@import "./abstracts/variables";
+
+// Bases
+@import "./bases/typography";
+@import "./bases/reset.scss";
```

Comment:
File name is not correct

Reply:
Fixed.

Reply:
Why we need scss extension here?

Reply:
Sorry for my carelessness, I will fix them.